keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/ChiSquared.java

```diff
@@ -8,34 +8,44 @@
 import io.improbable.keanu.tensor.intgr.IntegerTensor;
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
+/**
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier,
+ * ARL-TR-2168 March 2000,
+ * 5.1.4 page 16"
+ */
 public class ChiSquared implements ContinuousDistribution {
 
     private static final double LOG_TWO = Math.log(2);
-    private final IntegerTensor k;
+    private final IntegerTensor alpha;
```

Comment:
Wikipedia calls it `k`
Wolfram calls it `r`
Who calls it `alpha`?
We could call it `degreesOfFreedom` to be more descriptive

Reply:
The idea was to name shape parameters with greek alphabets to avoid clashing with tensor's `shape`.

Reply:
OK - it sounds like a bad idea to choose arbitrary greek letters.